experimental/SetPartitions/src/SetPartition.jl

```diff
@@ -404,3 +404,109 @@ function compose_count_loops(p::SetPartition, q::SetPartition)
     
     return (ret, length(related_comp))
 end
+
+"""
+    number_of_blocks(V::SetPartition)
+
+Return the number of blocks of a `SetPartition`.
+
+# Examples
+```jldoctest
+julia> number_of_blocks(SetPartition([1, 2, 3], [2, 1, 3, 3]))
+3
+```
+"""
+function number_of_blocks(V::SetPartition)
+    # obtain one vector describing the partition V
+    vec = vcat(upper_points(V), lower_points(V))
+
+    # return the maximum number in vec
+    if length(vec) != 0
+        return maximum(vec)
+    else
+        return 0
+    end
+end
+
+"""
+    is_dominated_by(V::SetPartition, W::SetPartition)
+
+Check if the set partition `V` is dominated by the set partition `W`. This is the case if every block of `V`
+is contained in exactly one block of `W`.
+
+# Examples
+```jldoctest
+julia> is_dominated_by(SetPartition([1, 1, 2], [1, 2, 3]), SetPartition([1, 1, 2], [1, 2, 1]))
```

Comment:
```suggestion
julia> is_dominated_by(set_partition([1, 1, 2], [1, 2, 3]), set_partition([1, 1, 2], [1, 2, 1]))
```
use the lowercase constructor in doctests